Framework/PythonInterface/plugins/algorithms/LoadEXED.py

```diff
@@ -0,0 +1,189 @@
+from __future__ import (absolute_import, division, print_function)
+from mantid.kernel import *
+from mantid.api import *
+
+import struct
+import numpy as np
+import copy
+import types
+
+
+def read_file(fn):
```

Comment:
Is there a reason not to move these functions to be part of the `LoadEXED` class?